Rubberduck.Parsing/Symbols/AccessibilityCheck.cs

```diff
@@ -4,94 +4,146 @@ public static class AccessibilityCheck
     {
         public static bool IsAccessible(Declaration callingProject, Declaration callingModule, Declaration callingParent, Declaration callee)
         {
-            if (callee.DeclarationType.HasFlag(DeclarationType.Project))
+            if (callee == null)
+            {
+                return false;
+            }
+            else if (callee.DeclarationType.HasFlag(DeclarationType.Project))
```

Comment:
superfluous else. or rather, why not `return callee != null && callee.DeclarationType.HasFlag(DeclarationType.Project);`?

The other blocks in this if-else chain can also be integrated into one boolean expression. Since this is customary in large portions of the project anyways I don't see the need to write them out so far in the first place